src/tribler-gui/tribler_gui/event_request_manager.py

```diff
@@ -56,16 +57,17 @@ def __init__(self, api_port, api_key):
             NTFY.LOW_SPACE.value: self.low_storage_signal.emit,
             NTFY.REMOTE_QUERY_RESULTS.value: self.received_remote_query_results.emit,
             NTFY.TRIBLER_SHUTDOWN_STATE.value: self.tribler_shutdown_signal.emit,
-            NTFY.EVENTS_START.value: self.events_start_received,
-            NTFY.TRIBLER_STARTED.value: lambda data: self.tribler_started.emit(data["version"]),
+            NTFY.EVENTS_START.value: lambda _: None,
+            NTFY.TRIBLER_STARTED.value: self.tribler_started_event,
         }
 
-    def events_start_received(self, event_dict):
-        if event_dict["tribler_started"]:
-            self.tribler_started.emit(event_dict["version"])
+    def tribler_started_event(self, event_dict):
+        self.tribler_started.emit(event_dict["version"], event_dict["public_key"])
```

Comment:
I've changed the event in which `tribler_starter` is emitted. 
@ichorid please, verify the correctness.

Reply:
Actually, the startup procedure is pretty tricky in regards to `EVENTS_START`-`TRIBLER_STARTED` interplay. The tricky part is the upgraded...

To be sure the thing will not break the upgrader, you'll have to test Tribler upgrading the database, say from 7.5->7.6 *and* trying to skip the upgrade.

Reply:
Mmm... I didn't think it is so complex.
Better to revert back then.

Reply:
It is only complex in regards to the `EVENTS_START` signal. Please, just test it with the upgraded and see if it works. If it does, keep your changes. I remember trying to remove the `EVENTS_START` signal altogether but failed due to the upgraded shenanigans. Maybe you'll have better luck with it :four_leaf_clover: :wink: 

If you don't want to risk it, you could still keep you `public_key`-passing change and just revert the `EVENTS_START`-related stuff.

Reply:
It seems to be working.